collectors/diskspace.plugin/plugin_diskspace.c

```diff
@@ -254,7 +254,7 @@ static inline void do_disk_space_stats(struct mountinfo *mi, int update_every) {
                                               netdata_zero_metrics_enabled == CONFIG_BOOLEAN_YES))) {
         if(unlikely(!m->st_space)) {
             m->do_space = CONFIG_BOOLEAN_YES;
-            m->st_space = rrdset_find_bytype_localhost("disk_space", disk);
+            m->st_space = rrdset_find_active_bytype_localhost("disk_space", disk);
```

Comment:
There are other places where `rrdset_find_bytype_localhost` is used. Shouldn't be updated there as well?

Reply:
But of course!

Reply:
It's possible my brain deleted MacOS for known reasons.